.ci/config/stage-types.yaml

```diff
@@ -20,9 +20,9 @@
 #     h2: [INT; OPTIONAL;]
 #     db2: [INT; OPTIONAL;]
 #     sqlserver: [INT; OPTIONAL;]
-engine-unit:
-  directory: 'engine'
-  command: 'clean test -P'
+db-unit:
+  directory: '.'
+  command: 'clean test -pl "engine,webapps" -Dskip.frontend.build=true -P'
   stash:
     runtimeStash: true
   labels:
```

Comment:
The `webapp-unit` stage also runs on  PRs with the `default-build` label. This label hasn't been included in the new `db-unit` stage type. Is this intentional?

Reply:
We're okay with this, as the `all-db` label can be set to execute these stages.